tests/nnstreamer_filter_python3/runTest.sh

```diff
@@ -37,13 +37,11 @@ if [[ -d $PATH_TO_PLUGIN ]]; then
     fi
 else
     echo "No build directory"
-    report
```

Comment:
It seems that you only want to test the nnstreamer without building.
But I think these lines are necessary.
How about changing like this?
```
# Default installation path. If you installed it somewhere else, change it.
INSTALL_PATH=/usr/lib/nnstreamer/filters/
check=$(ls ${INSTALL_PATH} | grep python3.so)
if [[ ! $check ]]; then
    echo "Cannot find python3 shared lib"
    report
    exit
fi
```